demos/common/python/openvino/model_zoo/model_api/models/classification.py

```diff
@@ -23,11 +23,11 @@
 
 
 class Classification(ImageModel):
-    __model__ = 'Classification'
+    __model__ = 'ote_classification'
```

Comment:
I believe these changes break classification demo, which uses this model wrapper.
Why do you need 2 additional outputs?
Can we just relax check of outputs if prediction behavior is not changed, just allow ignoring unused for getting classification results outputs?

Reply:
Why do you need 2 additional outputs? 
=> by this error : `openvino.model_zoo.model_api.models.model.WrapperError: None: There is no model with name "ote_classification" in list: bert, bert-named-entity-recognition, bert-embedding, bert-question-answering, Robust-video-matting, Background-matting, centernet, DETR, CTPN, FaceBoxes, RetinaFace, RetinaFace-PyTorch, SSD, Ultra_LightWeight_Face_Detection, YOLO, YOLOV4, YOLOF, YOLOX, YOLOv3-ONNX, Deblurring, HPE-assosiative-embedding, MaskRCNN, Yolact, Segmentation, Salient_Object_Detection, MonoDepth, OpenPose ` 

Can we just relax check of outputs if prediction behavior is not changed, just allow ignoring unused for getting classification results outputs?
=> Yes, I'm OK with your suggestion. Thank you!

Reply:
Wrapper error explains only why you tried to rename `__model__`, but my question about this check:
`self._check_io_number(1, 3)`.
I'm not sure that renaming on OMZ side is good idea, because there is nothing changed from original classification wrapper, just small check which can be generalized to accept both 1 and 3 outputs if it is necessary.

Reply:
The reason why I changed to get 3 outputs is from here : https://github.com/openvinotoolkit/model_preparation_algorithm/blob/develop/mpa/modules/models/classifiers/sam_classifier.py#L276, OTE classifier outputs (output_logit, feature_vector, saliency_map) when we export the model. 

For `ote_classification`, I agree with your suggestion is more reasonable for me, I'll review OTE side code to fit `ote_classification` -> `classification`

Reply:
Hi @eaidova, I think OTE need new template instead of renaming `__model__`. 
OTE is using OteClassification which inherits Classification class and overrides quiet many methods : https://github.com/openvinotoolkit/training_extensions/blob/develop/external/deep-object-reid/torchreid_tasks/model_wrappers/classification.py#L30

Can I create new file like `blur.py` for OTE classification? How do you think about this?

Reply:
@JihwanEom it is up to your, generally, we prefer to have all wrappers for one task is one file (ote_classification is still classification, is not it?), exception from this rule is case when model class represent specific topology or family (e.g. yolo)